definitions/pipelines/bisulfite.cwl

```diff
@@ -30,6 +30,12 @@ inputs:
         type: int
     QCannotation:
         type: File
+#nonmethylation_sites_bed is a variable to determine if user wants to obtain obtain bed/bigwig files for non-CpG cytosines. Value has to be true or false.
```

Comment:
This comment looks like a prime candidate to convert to a `doc` line for the parameter it goes to.  (I don't see a `nonmethylation_sites_bed` here.  Is it for `assay_no_cpg_sites`?)

Reply:
Changed it